package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "definitely-typed",
-    "version": "0.0.3",
+    "version": "0.0.4",
```

Comment:
Please don't make changes to the repo-wide package.json.

Reply:
Alright, I've just reverted that change now.